lstchain/calib/camera/pixel_threshold_estimation.py

```diff
@@ -62,17 +62,18 @@ def get_threshold_from_dl1_file(dl1_path, sigma_clean):
     threshold_clean_pe = ped_mean_pe + sigma_clean*ped_std_pe
     # find pixels with std = 0 and mean = 0 <=> dead pixels in interleaved
     # pedestal event likely due to stars
-    unusable_pixels = get_unusable_pixels(dl1_path, interleaved_events_id)
+    unusable_pixels = get_unusable_pixels(dl1_path)
     # for dead pixels set max value of threshold
     threshold_clean_pe[interleaved_events_id, HIGH_GAIN, unusable_pixels] = \
         max(threshold_clean_pe[interleaved_events_id, HIGH_GAIN, :])
     # return pedestal interleaved threshold from data run for high gain
     return threshold_clean_pe[interleaved_events_id, HIGH_GAIN, :]
 
-def get_unusable_pixels(dl1_path, interleaved_events_id):
+def get_unusable_pixels(dl1_path):
     with tables.open_file(dl1_path) as f:
+        calibration_id = f.root.dl1.event.telescope.monitoring.pedestal.col('calibration_id')
         unusable_pixels = np.where(f.root[dl1_params_tel_mon_cal_key].col(
-                                   'unusable_pixels')[interleaved_events_id,
+                                   'unusable_pixels')[calibration_id[1],
```

Comment:
Why the hardcoded 1 here?

Reply:
as I understood `1` in `calibration_id = f.root.dl1.event.telescope.monitoring.pedestal.col('calibration_id')` correspond to id for interleaved events.

Reply:
Then please define a constant `INTERLEAVED_CALIBRATION_ID`

Reply:
Done